client/my-sites/plans/main.jsx

```diff
@@ -31,7 +31,7 @@ const Plans = React.createClass( {
 	getDefaultProps() {
 		return {
 			intervalType: 'yearly',
-			displayJetpackPlans: true
+			displayJetpackPlans: false
```

Comment:
If there's this much confusion let's pull this out to 2 explicit components Jetpack vs WPCOM plans in a future PR. There isn't much interesting going on in this setup